### PR TITLE
Create fast-food-classic.json

### PR DESCRIPTION
**Pull request by ryansperzel**

### CODE REVIEW

**Review by alfg — CHANGES_REQUESTED** [review]
Missing image and invalid json.

**Comment by ryansperzel** [comment]
@alfg should be all good now

File: src/recipes/fast-food-classic.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        }
+    ],
+    "directions": [
+        "Make your fast food order a meal so it comes with a drink",
+        "Create the perfect blend of sodas at the neverage station. Don't be afraid to stray from the recipe and try new things",
+        "Drink at least 1/3 of it at the beverage station and fill it back up",
+        "Enjoy!",
+        "Optionally, head back for a refill or try a new blend!"
+    ],
+    "image": "fastfoodclassic.jpg",
```
Comment (alfg) [inline src/recipes/fast-food-classic.json:34]:
```suggestion
    "image": "fast-food-classic.jpg",
```

Can you update the image filename to use hyphens like the json file? Thanks.

**Review by alfg — CHANGES_REQUESTED** [review]

**Comment by ryansperzel** [comment]
@alfg Done and done

File: src/recipes/fast-food-classic.json (hunk 1 above)
Comment (ryansperzel) [reply src/recipes/fast-food-classic.json]: @alfg Done and done

**Comment by alfg** [comment]
Thanks for fixing!

**Pull request merged**